fix gcd

romatsp opened the pull request:
Fixed GCD function

romatsp committed:
fix gcd

yevhentrofimenko commented on gcd.py:
```diff
@@ -1,13 +1,13 @@
-def gcd(a, b):
-    assert a <= 0 and b >= 0
+def gcd(a: int, b: int):       #1
+    assert a < 0 and b > 0     #2
```
In task I mentioned that "a" and "b" must be positive, in your case: "a" can be less and 0 is not included, that's incorrect for requirements that I wrote.

romatsp committed:
So of course, my mistake. For some reason, I thought that the assert is an
output, not a condition (assert <condition>, <error message>).
I did a lot of checks, but did not expect a<0. I will be more attentive.
(From the words of Google Translator - it's recorded correctly, I hope :)

romatsp committed:
:( deleted comma from assert
